### PR TITLE
Doc: Updates to site style and addition of vale

### PR DESCRIPTION
**Pull request by JulioPDX**
Added or corrected the following:

- Add Arista favicon icon to site
- Minor corrections on spacing and spelling
- Add minor CSS for site theme
- Enabled social icon for GitHub and Global site
- Enabled code block highlighting

### CODE REVIEW

**Comment by JulioPDX** [comment]
Please let me know if Vale is a little much, I can always remove it. Thank you

**Review by sulrich — CHANGES_REQUESTED** [review]
- candidly, i think the vale addition is a (bit of a) reach too far. we've had enough challenges getting folks to pass the standard linting processes.  though the integration checks have helped enforcement on this.  grammar/style checks seem almost punitive. 😉 
- love the style updates

File: docs/examples/gnmi-clients/gnmic/index.md [hunk 1]
```diff
@@ -74,13 +74,13 @@ gnmic -a 192.0.2.214:6030 -u cvpadmin -p arista --insecure --gzip get --path "/"
...
 ```shell
 gnmic -a 192.0.2.214:6030 -u cvpadmin -p arista --insecure --gzip get --path  \
-  '/network-instances/network-instance[name=default]/protocols/protocol[identifier=BGP][name=BGP]/bgp'`
+  '/network-instances/network-instance[name=default]/protocols/protocol[identifier=BGP][name=BGP]/bgp'
 ```
 
 #### Get BGP neighbors
 
 ```shell
-gnmi -addr 192.0.2.139 -username admin get \
```
[the remark this answers is not in the document]
Comment (JulioPDX) [reply docs/examples/gnmi-clients/gnmic/index.md:83]: That would be good. Makes it easier for the user running through the examples. Vs editing each command for their environment.

**Comment by JulioPDX** [comment]
@sulrich Just removed the vale additions. Thank you for the feedback

**Review by sulrich — APPROVED** [review]

**Pull request merged**